uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/PublicHolidayLeaveRequestCreation.php

```diff
@@ -359,6 +359,40 @@ public function createAllForAbsencePeriod($absencePeriod, array $contactID = [])
     }
   }
 
+  /**
+   * Creates Public Holiday Leave Requests for all Public Holidays in all
+   * absence periods for all contacts. It does not re-create a leave request
+   * that already exists.
+   *
+   * The Public Holiday leave request will not be created if a contact has no
+   * entitlement for the absence type with MTPHL in the absence period the
+   * public holiday date falls in or If the contact does not have a contract
+   * overlapping the public holiday date.
+   *
+   */
+  public function createForAll() {
```

Comment:
I  was checking the methods names in this class we have have two patterns: those starting with `createAllFor` and those with `createForAll`. The former, creates requests for all Public Holidays for a specific entity (Absence Periods, Contacts, Contracts, WP). The latter, creates requests for a single Public Holiday but for all the records of specific entity set (for all Absence Periods, for all contracts, etc).

Given that, I think we should rename this method to `createAll()`. Also, to make this consistent, I think we should rename `createForAllInTheFuture` to `createAllInTheFuture` (Also, note that the respective method in the Deletion logic class is named `deleteAllInTheFuture` instead of `deleteForAllInTheFuture`)